tests/base.py

```diff
@@ -285,29 +286,34 @@ def setUp(self):
 class SgTestConfig(object):
     '''Reads test config and holds values'''
     def __init__(self):
-        self.mock           = True
-        self.server_url     = None
-        self.script_name    = None
-        self.api_key        = None
-        self.http_proxy     = None
-        self.session_uuid   = None
-        self.project_name   = None
-        self.human_name     = None
-        self.human_login    = None
-        self.human_password = None
-        self.asset_code     = None
-        self.version_code   = None
-        self.shot_code      = None
-        self.task_content   = None
 
+        for key in self.config_keys():
+            value = None
+            if key in ['mock']:
+                value = True
+            # Look for any environment variables that match our test
+            # configuration naming of "SG_{KEY}". Default is None.
+            value = os.environ.get('SG_{0}'.format(str(key).upper()))
```

Comment:
I'm too cool for school apparently-- will update that to % syntax.
As well @pboucher mentioned to do a check for whether "mock" is being passed in as an env var and change that to a boolean.
Will update those now.